src/FINDOLOGIC/Export/XML/XMLItem.php

```diff
@@ -153,7 +164,7 @@ private function buildUsergroups(\DOMDocument $document): \DOMElement
      * @param array $images The images to validate.
      * @return boolean Whether the images are valid or not.
      */
-    private function validateImages(array $images): bool
+    private static function validateImages(array $images): bool
```

Comment:
Why using a static method here?

Reply:
See [this comment](https://github.com/findologic/libflexport/pull/98#discussion_r267884157).